src/script/calling/CallingRepository.ts

```diff
@@ -2435,6 +2500,13 @@ export class CallingRepository {
     PrimaryModal.show(PrimaryModal.type.ACKNOWLEDGE, modalOptions);
   }
 
+  private isSelfUser(participant: Participant): boolean {
+    if (this.selfUser == null || this.selfClientId == null) {
+      return false;
+    }
+    return participant.doesMatchIds(this.selfUser.qualifiedId, this.selfClientId);
+  }
```

Comment:
we already have the "matchQualifiedIds" helper function that is used for selfUser checks, we could use this here as well just to be aligned